Scripts/script-HTTPListRedirects.yml

```diff
@@ -0,0 +1,40 @@
+commonfields:
+  id: HTTPListRedirects
+  version: -1
+name: HTTPListRedirects
+script: |-
+  import requests
+
+  response = requests.head(demisto.args()['url'], allow_redirects=True) if demisto.args()['useHead'] == 'true' else requests.get(demisto.args()['url'])
+  urls = []
+  if response.history:
+      for resp in response.history:
+          urls.append({'Details': resp.url, 'Status': resp.status_code})
+  urls.append({'Details': response.url, 'Status': response.status_code})
+  ec = {'URL(val.Details == obj.Details)': [{'Details': u['Details']} for u in urls]}
+
+  demisto.results({'ContentsFormat': formats['json'], 'Type': entryTypes['note'], 'Contents': urls, 'ReadableContentsFormat': formats['markdown'],
+      'HumanReadable': tableToMarkdown('URLs', urls, ['Details', 'Status']), 'EntryContext': ec})
+type: python
+tags:
+- Utility
+comment: List the redirects for a given URL
+enabled: true
+args:
+- name: url
+  required: true
+  default: true
+  description: The URL to follow
+- name: useHead
+  auto: PREDEFINED
+  predefined:
+  - "true"
+  - "false"
+  defaultValue: "true"
+  description: Should we use HEAD instead of GET. Safer but might not be the same
+    response.
+outputs:
+- contextPath: URL.Details
+  description: The URL redirects from the given URL
+scripttarget: 0
+releaseNotes: "New script to list redirects of URL"
```

Comment:
Remove release notes
They are only for modified scripts/scripts - for new scripts that you dont want to add to release notes